### PR TITLE
Minor updates to iD editor learning module writing style, plus a few …

### PR DESCRIPTION
**Pull request by gitGOwin**
…changes from UK English to US English spelling.

### CODE REVIEW

File: _posts/en/editing/0300-10-31-id-editor.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ category: editing
 
 The iD Editor
 =============
-The iD editor is the new (launched 2013) web-based OpenStreetMap editor which makes it easy to edit OpenStreetMap. iD is fast and easy, and allows you to map using various data sources such as satellite imagery, GPS, and Field Papers.
+The iD editor is the new (launched 2013) web-based OpenStreetMap editor which makes it easy to edit OpenStreetMap. iD is fast and easy, and allows mapping of various data sources such as satellite imagery, GPS, ,Field Papers and Mapillary.
```
Comment (althio) [inline _posts/en/editing/0300-10-31-id-editor.md]: l.11 - allows mapping of various data sources such as satellite imagery, GPS, ,Field Papers and Mapillary.
l.11 + allows mapping from various data sources such as satellite and aerial imageries, GPS, Field Papers or Mapillary.

File: _posts/en/editing/0300-10-31-id-editor.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ category: editing
...
 
-The iD editor is a great way to edit when you want to make small and easy changes, and you don't require all the complexity of JOSM. This chapter demonstrates the basics of editing with iD. 
+The iD editor is a great way to edit for small and easy changes that don't require the advanced features of JOSM. This chapter demonstrates the basics of editing with iD. 
 
 Starting the iD Editor
 ----------------------
--	To use iD you will need to be connected to the internet.
--	Open your internet browser, and go to the OpenStreetMap website at [http://www.openstreetmap.org](http://www.openstreetmap.org).
+-	To iD editor requires an active connection to the Internet.
```
Comment (althio) [inline _posts/en/editing/0300-10-31-id-editor.md]: l.17 - To iD editor requires
l.17 + The iD editor requires

**Comment by Nick-Tallguy** [comment]
@althio can I just check that you're managing this update. 

Regards

Nick

**Comment by althio** [comment]
@Nick-Tallguy I made some inline comments and allow some time to @gitGOwin to respond.

**Comment by gitGOwin** [comment]
Sorry, I'm a bit of a noob with your workflow. @althio , we're you expecting me to make the necessary revisons for the suggestions you made? 

And after those revisions, are you expecting another pull request?

**Comment by gitGOwin** [comment]
@althio @Nick-Tallguy 
Sorry for the hold-up. I've incorporated the edits suggested by @althio (thanks!)

**Comment by althio** [comment]
@gitGOwin No worries, sorry for being unclear with my notes!
You did it totally alright, new commit, same pull request, that is fine by me.

Out of curiosity, what would be UK English vs. US English spelling here?

**Pull request merged**